packages/mask/shared-ui/locales/en-US.json

```diff
@@ -1136,7 +1136,7 @@
     "popups_persona_connect_to": "Connect to {{type}}",
     "popups_persona_to_be_verified": "To be verified",
     "popups_persona_disconnect": "Disconnect",
-    "popups_persona_disconnect_confirmation": "Confirm disconnect",
+    "popups_persona_disconnect_confirmation": "Confirm disconnect?",
```

Comment:
?

Reply:
Just for design

<img width="561" alt="image" src="https://user-images.githubusercontent.com/28348152/191510990-d367a534-22c1-458a-9571-f238bf66ae6c.png">